AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2137,6 +2137,18 @@ - (NSString *)_recursiveDescriptionHelperWithIndent:(NSString *)indent
   return subtree;
 }
 
+#pragma mark - ASLayoutableAsciiArtProtocol
+
+- (NSString *)asciiArtString
+{
+    return [ASLayoutSpec asciiArtStringForChildren:@[] parentName:[self asciiArtName]];
```

Comment:
This should just print the node's name. If someone wanted to make ascii art of the node hierarchy they'd pass in self.subnodes as the children.